dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/LengthFieldStreamingDecoder.java

```diff
@@ -167,8 +160,8 @@ private void processBody() throws IOException {
         requiredLength = lengthFieldOffset + lengthFieldLength;
     }
 
-    protected void invokeListener(InputStream inputStream) {
-        this.listener.onFragmentMessage(dataHeader, inputStream);
+    public void invokeListener(InputStream inputStream) {
+        this.listener.onFragmentMessage(inputStream);
```

Comment:
**Security Issue**: Changing the visibility of the invokeListener method from protected to public without proper validation can expose internal processing mechanisms to unintended usage, potentially leading to unauthorized access or denial of service. <br> **Fix**: Reinstate the method's protected visibility and ensure that any external calls to this method are properly authenticated and validated to prevent unauthorized access. <br> **Code Suggestion**: 
 ```
 -    protected void invokeListener(InputStream inputStream) {
 +    private void invokeListener(InputStream inputStream) {
 ```